modules/checkout/templates/commerce-checkout-order-summary.html.twig

```diff
@@ -0,0 +1,28 @@
+{#
+/**
+ * See commerce-order-receipt.html.twig for examples.
+ *
+ * Available variables:
+ * - order_entity: Order entity
+ * - items: The order items
+ * - totals: Result from commerce_order.order_total_summary service
+ * - checkout_flow_plugin_id: The current checkout flow ID.
```

Comment:
See my comment above

> These are here so that people can do conditionals in the Twig template. For instance, we use this to check if on "order_information" and display "+ applicable taxes" if there are no adjustments. Should we document that somehow here, or in the Twig template?

I use them often; we injected them into our own templates.